app/views/idv/phone/new.html.erb

```diff
@@ -1,3 +1,11 @@
+<div id="form-steps-wait-alert">
+  <%= render 'shared/alert', {
+    type: 'success',
+    class: 'margin-bottom-4',
+    message: I18n.t('doc_auth.forms.doc_success'),
```

Comment:
> this is always success because took this out of `verify_wait_step_show` right?

Right, instead of assigning by flash, we statically show the alert on the following step. This makes an assumption that the only case for reaching this view is via successful completion of the `verify` step.